certificate-policy.md

```diff
@@ -108,7 +108,7 @@ The primary goal of these Requirements is to enable efficient and secure electro
 
 
 ### 1.4.2 Prohibited Certificate Uses
-All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption. Trusted Role certificates MAY be issued from either the Root or Subordinate CA.
+All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption unless issued for specific infrastructure purposes as outlined in section 6.1.7.
```

Comment:
CA products may issue people certificates to authenticate users to the software. Number 3 in the list "administrator role certificates" is an example. They are internal only certificates. They aren't meant to be used for any other purpose outside of the CA or to be discovered / logged publicly. Trusted Roles are outlined in section 5.2.1 which is blank at the moment.